app/javascript/Pages/Connections.jsx

```diff
@@ -4,35 +4,29 @@ import { Head, Link } from "@inertiajs/inertia-react";
 import "../stylesheets/body.css";
 
 export default function Welcome(props) {
-  const token = document.querySelector(
-    "[name=csrf-token]"
-  ).content;
+  const token = document.querySelector("[name=csrf-token]").content;
```

Comment:
**question (blocking)**: is this needed with the change you made to save this in the window?